tests/MagicMethodAnnotationTest.php

```diff
@@ -789,4 +789,31 @@ class B extends A {}
             ],
         ];
     }
+
+    /**
+     * @return void
+     */
+    public function testSealAllMethods()
+    {
+        Config::getInstance()->seal_all_methods = true;
+
+        $this->addFile(
```

Comment:
Can I add more files in the same method? There are a number of other cases I'd like to test.

Reply:
Just create separate tests!